tk205/file_io.py

```diff
@@ -1,39 +1,44 @@
 import os
 import json
 import cbor2
+import yaml
 from .xlsx import template, A205XLSXTree
 
 def get_extension(file):
     return os.path.splitext(file)[1]
 
-def load(file):
-    # todo check file exisits
-    ext = get_extension(file).lower()
+def load(input_file_path):
+    ext = get_extension(input_file_path).lower()
     if (ext == '.json'):
-        with open(file, 'r') as input_file:
+        with open(input_file_path, 'r') as input_file:
             return json.load(input_file)
     elif (ext == '.cbor'):
-        with open(file, 'rb') as input_file:
+        with open(input_file_path, 'rb') as input_file:
             return cbor2.load(input_file)
     elif (ext == '.xlsx'):
         tree = A205XLSXTree()
-        return tree.load_workbook(file).get_content()
+        return tree.load_workbook(input_file_path).get_content()
+    elif (ext == '.yaml'):
+        return yaml.load(input_file_path, Loader=yaml.FullLoader)
```

Comment:
I tried `return yaml.load(input_file_path)` and it works just fine. Do we really need to do something different with the `Loader`? Do we know what this is even doing?

Reply:
Using the load function without a Loader is apparently unsafe, and functionality for making this call without a loader is deprecated and throws a warning. In the wiki they explain that this call has been found to be exploited. The "FullLoader" is the recommended default, but I don't think it is functionally doing anything different. You can read about it here if you want full details: 
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation. 

I believe that this call is meant to be replaced by the "safe_load()" function in future releases, but it currently only handles a subset of yaml, not the full language (that is the way I read the description in the docs) so I opted for this call. We could consider switching to it.

Reply:
Okay, let's keep the Loader.